django/views/debug.py

```diff
@@ -70,6 +70,12 @@ def get_safe_settings():
     return settings_dict
 
 
+def get_safe_request_meta(request):
+    """Return a dictionary of request.META with sensitive values redacted."""
+    has_meta = hasattr(request, 'META')
+    return {k: cleanse_setting(k, v) for k, v in request.META.items()} if has_meta else {}
```

Comment:
Originally it was:

```python
if request and hasattr(request, 'META'):
    return {k: cleanse_setting(...
return {}
```

and @timgraham suggested I change to the above. I am fine with any style, let me know what is prefered